packages/core/src/Properties.ts

```diff
@@ -19,21 +19,28 @@ export class Properties {
       return this._currentLanguage;
     }
 
+    let result: string | undefined;
```

Comment:
Don't use the result variable. Instead, return assign the this._currentLanguage before returning the result in each branch. I am looking into the method like 5 minutes now and I still don't really get it. 😆

Reply:
Maybe it's better to put it in saparate method